public/app/core/components/sidemenu/TopSectionItem.tsx

```diff
@@ -21,7 +21,7 @@ const TopSectionItem: FC<Props> = ({ link, onClick }) => {
       {linkContent}
     </Link>
   ) : (
-    <a className="sidemenu-link" onClick={onClick}>
+    <a className="sidemenu-link" onClick={onClick} href="">
```

Comment:
If a link has an empty href, should it really be a link? All the accessibility recommendations I see online suggest using a button instead of `a` if there's no `href` or it's empty. We have `link` variant for buttons, which I think should work here.

Reply:
I was thinking about this too, but came to the conclusion that it behaves like a link. At least in the search case. It's a possibility that I plugin would want to do something different.

Maybe building the entire search URL would be a better option and then actually use it as a link?

Reply:
I think to really behave like a link, it'd have an href, smth tells me the screen readers aren't gonna like it as it is now.

Reply:
I tried it with VoiceOver, and it works pretty well. I think the client side routing does some weird stuff in general, so the other links don't work that well (you click the link but nothing is announced). But do you think constructing a URL would be a better solution than making it a button?

Reply:
Yeah, I think it'd be a better approach but also I'm not sure if it'll break its application in other places as it seems like this is a generic component.